pkg/workflows/wasm/workflowLib/factory.go

```diff
@@ -0,0 +1,106 @@
+package workflowLib
+
+import (
+	"context"
+	"time"
+
+	"google.golang.org/protobuf/proto"
+	"google.golang.org/protobuf/types/known/durationpb"
+
+	"github.com/smartcontractkit/chainlink-common/pkg/logger"
+	"github.com/smartcontractkit/chainlink-common/pkg/services"
+	"github.com/smartcontractkit/chainlink-common/pkg/workflows/wasm/workflowLib/pb"
+	"github.com/smartcontractkit/libocr/offchainreporting2plus/ocr3types"
+)
+
+const (
+	defaultMaxPhaseOutputBytes     = 1000000 // 1 MB
+	defaultMaxReportCount          = 20
+	defaultBatchSize               = 1000
```

Comment:
This can be much larger, maybe 10000? Each request is a 64 byte string and a number.